legal-api/src/legal_api/reports/registrar_meta.py

```diff
@@ -52,19 +52,11 @@ class RegistrarInfo:   # pylint: disable=too-few-public-methods
             'endDate': '2025-04-17T23:59:59'
         },
         {
-            'name': "S. O\'CALLAGHAN",
```

Comment:
This is an old error but we weren't doing anything with the name previously.

Now, I check the name in a unit test, so I had to fix this.